runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -204,6 +221,30 @@ public void onGameTick(GameTick tick)
 			return;
 		}
 
+		for (DriftNet net : NETS)
+		{
+			DriftNetStatus netStatus = net.getStatus();
+			DriftNetStatus prevTickNetStatus = net.getPrevTickStatus();
+			if (isAcceptingFish(netStatus) || isAcceptingFish(prevTickNetStatus))
+			{
+				Iterator<NPC> itr = taggedFish.keySet().iterator();
+				// untag fish which are next to a net that is not accepting fish
+				while (itr.hasNext())
+				{
+					NPC singleTaggedFish = itr.next();
+					WorldPoint fishPoint = WorldPoint.fromLocalInstance(client, singleTaggedFish.getLocalLocation());
+					for (WorldPoint point : net.getAdjacentTiles())
```

Comment:
I think it would be better if `adjacentTiles` was just a set, which would let you do `adjacentTiles.contains(fishpoint)`